CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - wrong order of arguments in fire work effects
 - non-static `variables` causing cross conversation validation to break
 - LuckPerms integration not pushing the permission update to the connected servers correctly.
+- `crafting` fixed bug, where complex recipes were not recognized
```

Comment:
Try to write it like the sentence already starts with `Fixed` as it is the topic it is in.
```suggestion
- `crafting` objective where complex recipes are not recognized
```